koku/masu/processor/ocp/ocp_report_db_cleaner.py

```diff
@@ -74,8 +75,12 @@ def purge_expired_report_data(self, expired_date=None, provider_uuid=None, simul
 
                 all_report_periods.sort()
                 LOG.info(
-                    f"Removing all data related to report_period_ids: {all_report_periods}; "
-                    f"cluster_ids: {all_cluster_ids}; starting periods {all_period_starts}"
+                    log_json(
```

Comment:
I don't have a CLUE why we are _sorting_ before logging this! So let's remove that. Also, have a look here at what we've done for OCI and try align this file with that one:

https://github.com/project-koku/koku/pull/4513/files#diff-a3552de280b5b827a2a892b26024c8cb2ebc51f0006761df69cd45b2184e319cR76

Reply:
Maybe the sort was the keep the string consistent for searching? We can get remove that.

Reply:
I made an attempt to make this consistent with the logging in OCI. Not sure how close I got.

Reply:
This looks good, it just need to be moved under the `for` loop.